### PR TITLE
Remove Everything Bagel

### LINKED ISSUE #6144: remove Everything Bagel
- [x] Update both EB blogs ([1](https://lakefs.io/blog/the-docker-everything-bagel-spin-up-a-local-data-stack/) [2](https://lakefs.io/blog/the-everything-bagel-ii-versioned-data-lake-tables-with-lakefs-and-trino/)) to point users to https://github.com/treeverse/lakeFS-samples
- [x] rm -rf ./deployments/compose/*
- [x] Update ./deployments/compose/README.md to point users to https://github.com/treeverse/lakeFS-samples

### PR DESCRIPTION
**Pull request by rmoff**
### Linked Issue

Closes #6144

---

### Background

Everything Bagel has been migrated and incorporated into the lakeFS-samples repository. 

For the background to this please see https://github.com/treeverse/lakeFS/issues/6077.

### New Feature

This PR removes Everything Bagel from the lakeFS repo and replaces it with a notice directing users to the lakeFS-samples repo.

### CODE REVIEW

**Review by arielshaqed — APPROVED** [review]
Yay, thanks!

File: deployments/compose/301_moved.png
Comment (nopcoder) [inline deployments/compose/301_moved.png:1]: Do we reference the link to this folder on github? we don't need to an image as part of our repo?
Usually I don't need to render local markdown unless it is part of the docs.

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: deployments/compose/README.md [hunk 1]
```diff
@@ -1,120 +1,5 @@
-# lakeFS `docker-compose`
+# Everything Bagel
 
-Compose file for lakeFS using variety of services lakeFS can work with.
-Automatically creates a minIO bucket and lakeFS repository as part of the start up.
+![`301 Moved Permanently`](301_moved.png)
```
[the remark this answers is not in the document]
Comment (rmoff) [reply deployments/compose/README.md:3]: It's supposed to be a nerdy joke :P

File: deployments/compose/301_moved.png
Comment (rmoff) [reply deployments/compose/301_moved.png:1]: Given that it's linked to externally I wanted to make it a "friendlier" page for developers to land on and hopefully stay engaged with - hence the image.

Comment (rmoff) [reply deployments/compose/301_moved.png:1]: the folder isn't referenced anywhere else from the github repo that I can find.

**Review by nopcoder — APPROVED** [review]

**Pull request merged**